docs/configuration-reference-source.md

```diff
@@ -0,0 +1,103 @@
+# Database
```

Comment:
Suggest renaming file to `docs/configuration-reference-changes-source.md` to align with above suggestions about source type.